core/arch/aarchxx/mangle.c

```diff
@@ -74,12 +74,16 @@ typedef struct ALIGN_VAR(16) _icache_op_struct_t {
      * consecutive icache lines (which could wrap around the top of memory).
```

Comment:
Could we add some kind of test for this mangling?  A simple app that reads the DIC bit and prints it out or sthg and is run with the different options, in our regression suite so we won't accidentally break this and not notice in the future.